### ISSUE TITLE
Not compatible with PHP 7.*?

### ISSUE THREAD

**Issue by fvlthy** [frostschutz/MyBB-Google-SEO#63]
Hey, i attemped to install Google SEO but i kept getting warnings
<addr>

File | Line | Function
-- | -- | --
C:\xampp\htdocs\LeakZone\inc\plugins\google_seo\plugin.php | 550 | errorHandler->error
C:\xampp\htdocs\LeakZone\inc\plugins\google_seo\plugin.php | 68 | google_seo_plugin_status
C:\xampp\htdocs\LeakZone\inc\plugins\google_seo.php | 50 | google_seo_plugin_info
C:\xampp\htdocs\LeakZone\admin\modules\config\plugins.php | 527 | google_seo_info
C:\xampp\htdocs\LeakZone\admin\index.php | 821 | require
</addr>

I figured that this might be because i use PHP 7.2.1 but i cannot seem to find a XAMPP version with PHP 5.1.0?
,

**Comment by frostschutz** [comment]
Are you using the version from GitHub? It already has two fixes regarding PHP 7.2.x compatibility. There is also a fix in github frostschutz/MyBB-PluginLibrary.

I have to do a full code review regarding 7.2.x compatibility, it seems I've been checking arrays in particular in a way PHP does not like (anymore).

**Comment by fvlthy** [comment]
Which release of PluginLibrary do i get for MyBB 1.8.15? I tried 12 and 11 and MyBB says theyre not compatible, just downloaded the newest version of Google SEO from your github.

**Comment by Eldenroot** [comment]
Do you have any ETA? This plugin is a must have one, but my webhosting pushes and will update to php 7.2 in 12 days :(

**Comment by PennyLook** [comment]
@fvlthy @Eldenroot 
Here's fix:
you should change 550 line to:

` if($edits && count($edits) ==1)`

After that I do not have any warnings.

**Comment by frostschutz** [comment]
@PennyLook good catch, but it can also be > 1. I added the missing variable initialization.

**Comment by marble34534** [comment]
Hello,

If I use PHP 7.2, I will get the following error:

Type: 2
File: inc/plugins/google_seo/sitemap.php (Line no. 462)
Message
sizeof(): Parameter must be an array or an object that implements Countable

I use Google SEO (1.8.2) and MyBB 1.8.18

**Comment by frostschutz** [comment]
Thanks, I'll try to make a new release over the weekend. Remind me if I don't.

**Comment by frostschutz** [comment]
Released the update (along with an update for PluginLibrary).

Please let me know if there any more problems. Thank you.
